db/migrate/20221129063218_adjust_report_design_schema.rb

```diff
@@ -0,0 +1,30 @@
+class AdjustReportDesignSchema < ActiveRecord::Migration[5.2]
+  def change
+    RequestStore.store[:skip_rules] = true
+
+    at = DynamicAnnotation::AnnotationType.where(annotation_type: 'report_design').last
+    unless at.nil?
+      json_schema = at.json_schema.clone.with_indifferent_access
+      options_schema = json_schema[:properties][:options][:items]
+      json_schema[:properties][:options] = options_schema
+      
+      at.json_schema = json_schema
+      at.save!
+
+      # We need to do this in the migration, otherwise the app can be inconsistent
```

Comment:
In that case, let's schedule a deployment date/time where we can take the app down, @melsawy .

Reply:
I'll move this migration to rake task as we have around `227920` items to migrate and run it immediately after deployment.
and either to take app down or expect some user can not save a fact check for a while